users/api.py

```diff
@@ -64,3 +66,18 @@ def perform_update(self, serializer):
             serializer.save(password=make_password(password))
         else:
             serializer.save()
+
+
+class RegisterCustomerAPIView(APIView):
+    def post(self, request, format=None):
+        serializer = CustomerAccountSerializer(data=request.data)
+        if serializer.is_valid():
```

Comment:
please use "serializer.is_valid(raise_validation=True)". you don't need check password is found or not